DotNetEventPipe/Tables/GCTable.cs

```diff
@@ -0,0 +1,172 @@
+            new ColumnMetadata(new Guid("{CB84FD81-24EF-49AA-91B0-4BB2D00BBE0E}"), "Depth", "Depth"),
+            new UIHints { Width = 80 });
+
+        private static readonly ColumnConfiguration gcTypeColumn = new ColumnConfiguration(
+            new ColumnMetadata(new Guid("{096124B1-F46D-4204-8C0A-D4DA65ACEBDA}"), "Type", "GC Type"),
+            new UIHints { Width = 160 });
+
+        private static readonly ColumnConfiguration clrInstanceIDColumn = new ColumnConfiguration(
+            new ColumnMetadata(new Guid("{FFE7E4B2-0636-403B-A2F6-0B6D696C18FC}"), "ClrInstanceID", "Clr Instance ID"),
+            new UIHints { Width = 115 });
+
+        private static readonly ColumnConfiguration clientSequenceNumberColumn = new ColumnConfiguration(
+            new ColumnMetadata(new Guid("{105C1056-EC3B-4980-A36E-D0CBE1E4A479}"), "ClientSequenceNumber", "Client Sequence Number"),
+            new UIHints { Width = 145 });
+
+        private static readonly ColumnConfiguration timestampColumn = new ColumnConfiguration(
+            new ColumnMetadata(new Guid("{00746811-C2AE-48B8-AF00-1213AFFB4D6F}"), "Timestamp", "The timestamp of the event"),
+            new UIHints { Width = 80 });
+
+        private static readonly ColumnConfiguration durationColumn = new ColumnConfiguration(
+            new ColumnMetadata(new Guid("{2EFD88CE-0559-46D7-8E1A-43F5B2EDC482}"), "Duration", "Duration of the GC"),
+            new UIHints { Width = 80 });
+
+        private static readonly ColumnConfiguration threadIdColumn =
+            new ColumnConfiguration(
+                new ColumnMetadata(new Guid("{0CDD7D87-FDC3-457D-87CD-692EA4280664}"), "ThreadId"),
+                new UIHints { Width = 80, });
+
+        private static readonly ColumnConfiguration processColumn =
+            new ColumnConfiguration(
+                new ColumnMetadata(new Guid("{B3BA7352-86CC-4271-A136-04CA071D19C5}"), "Process"),
+                new UIHints { Width = 100, });
+
+        private static readonly ColumnConfiguration processIdColumn =
+            new ColumnConfiguration(
+                new ColumnMetadata(new Guid("{7F13B9C1-7FD5-417C-B768-637D7BF0CA55}"), "Process Id"),
+                new UIHints { Width = 80, });
+
+        private static readonly ColumnConfiguration cpuColumn =
+            new ColumnConfiguration(
+                new ColumnMetadata(new Guid("{5C9EAC74-23E2-4450-9CC0-65C516AC1422}"), "CPU"),
+                new UIHints { Width = 80, });
+
+
+        public override void Build(ITableBuilder tableBuilder)
+        {
+            if (TraceEventProcessor == null || TraceEventProcessor.Count == 0)
+            {
+                return;
+            }
+
+            var firstTraceProcessorEventsParsed = TraceEventProcessor.First().Value;  // First Log
+            var gcStartEvents = firstTraceProcessorEventsParsed.GenericEvents.Where(f => f.ProviderName == "Microsoft-Windows-DotNETRuntime" && 
```

Comment:
Pretty cool!  I would recommend sourcing this information from the state machine that knows how to build up the list of GCs instead of using GC/Start and GC/Stop events.  This will provide you with more information if it's available, but will also handle cases where you might have nested GC/Start/Stop events (e.g. in the case of background GCs where a foreground gen0 is triggered).  You can see where this gets created in PerfView here: [https://github.com/microsoft/perfview/blob/main/src/PerfView/PerfViewData.cs#L3532](https://github.com/microsoft/perfview/blob/main/src/PerfView/PerfViewData.cs#L3532)